mix.exs

```diff
@@ -49,7 +49,11 @@ defmodule GTFSRealtimeViz.Mixfile do
     [
       licenses: ["MIT"],
       links: %{"GitHub" => "https://github.com/mbta/gtfs_realtime_viz"},
-      maintainers: ["Gabe Durazo <gabe@durazo.us>", "John Kohler <kohlerjohnp@gmail.com>", "Alex Sghia-Hughes", "Dave Maltzan <dmaltzan@gmail.com>"]
+      maintainers: [
+        "Gabe Durazo <gabe@durazo.us>",
+        "Alex Sghia-Hughes",
+        "Dave Maltzan <dmaltzan@gmail.com>"
```

Comment:
I guess you might as well remove Dave here. And can you change my email address to my mbta one while you're at it?

Reply:
good points